.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
```

Comment:
via https://core.trac.wordpress.org/ticket/43828
> The patch for this ticket adds the ESLint JSDoc configuration to a file named .eslintrc-jsdoc.js, this is to avoid conflicts with the work in progress of adding ESLint in #31823

Nit, should we name this file `.eslintrc-jsdoc.js` instead, as I'd love to begin working on adding core JS ESlint next, and I'm not sure combining both ESLint rulesets would work well at this interim stage?

https://core.trac.wordpress.org/ticket/31823

Reply:
Wouldn't we want to combine to form a unified ruleset? I think we should bring it over like this, and then migrate later into a separate config if needed.

Reply:
I am making this change. We can combine if needed in the future.

Reply:
Sure, go for it 👍🏼